assets/scss/_k8s_search.scss

```diff
@@ -0,0 +1,22 @@
+.td-navbar, .td-search {
+  .td-search__input, .td-search__input::placeholder {
+    background: white;
+    color: black;
+  }
+
```

Comment:
```suggestion
  .td-search__input::placeholder {
    color: #888 !important;
  }
```

Reply:
I have made the search bar grey in fact to bring it in line with live. Making the elements grey now will be a11y worse imo.